ansible/install_jenkins.yml

```diff
@@ -0,0 +1,39 @@
+- hosts: localhost
+  become: true
+  gather_facts: true
+  
+  vars:
+    jenkins_hostname: localhost
+    java_packages:
+      - java-1.8.0-openjdk
+    
```

Comment:
add a variable for plugin install even if it's empty : jenkins_plugins: []

Reply:
corrected remarks